pkg/1596e275-314e-4bde-a5b5-11ea19d9d6e8/package.lp

```diff
@@ -3,7 +3,7 @@
  (description "Generic J-lead resistor 11569 (imperial 4527).\n\nLength: 11.56mm\nWidth: 6.98mm\n\nGenerated with librepcb-parts-generator (generate_chip.py)")
  (keywords "11569,4527,r,resistor,j-lead,generic")
  (author "Danilo B.")
- (version "0.3")
+ (version "0.3.1")
```

Comment:
Didn't we say we only use major.minor for the version?

Reply:
I'm not aware of that. My proposal is still to use only integers, and increment them by 1 on any change. Right now the decision whether to use major, minor or patch version is arbitrary, since semver doesn't make sense for parts.